runtime/master/src/main/java/edu/snu/nemo/runtime/master/RuntimeMaster.java

```diff
@@ -379,7 +378,6 @@ private void accumulateBarrierMetric(final List<ControlMessage.PartitionSizeEntr
 
   /**
    * Schedules a periodic DAG logging thread.
-   * TODO #58: Web UI (Real-time visualization)
```

Comment:
[Nemo-20](https://issues.apache.org/jira/browse/NEMO-20?jql=project%20%3D%20NEMO%20AND%20resolution%20%3D%20Unresolved%20AND%20text%20~%20%22restful%22%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC)?

Reply:
Addressed in 80eed5c